rbc/src/test/java/com/library/rbc/service/bookservice/BookServiceShould.java

```diff
@@ -48,4 +48,17 @@ void getAllBooks() {
     Page<BookDto> expectedBookDtos = createBookDtosPage();
     assertEquals(expectedBookDtos, result);
   }
+
+    @Test
+    void addNewBook() {
+        Book book = createBook();
+        BookDto bookDto = createBookDto();
+
+        when(bookMapper.bookDtoToBook(bookDto)).thenReturn(book);
+        when(bookRepository.save(book)).thenReturn(book);
+
+        bookService.addNewBook(bookDto);
+
+        assertEquals(bookDto, bookDto);
```

Comment:
Same comment for assertion part of the test. In this case you should compare the actual result of `bookService.addNewBook(bookDto)` with the expexted `bookDto`.